src/main/java/uk/gov/ons/ctp/response/notify/endpoint/StatusEndpoint.java

```diff
@@ -106,8 +106,7 @@ private Notification fetchNotificationFromNotifyService(final UUID notificationI
       return notifyService.findNotificationById(notificationId);
     } catch (NotificationClientException e) {
       String errorMsg = String.format(ERRORMSG_NOTIFICATION_ISSUE, e.getMessage(), e.getCause());
```

Comment:
Do we still need `e.getMessage(), e.getCause()`?

Reply:
well it goes into the exception that gets thrown

Reply:
But we're passing the whole exception through anyway `log.error(errorMsg, e);`

Reply:
Oh just realised it's getting used somewhere else, nm